test/codegen-generated-classes-test/src/test/java/software/amazon/awssdk/services/jmespath/JmesPathRuntimeValueTest.java

```diff
@@ -37,6 +38,15 @@
 import software.amazon.awssdk.utils.Pair;
 
 class JmesPathRuntimeValueTest {
```

Comment:
Did you test all of the new functionality in JmesPathRuntime? Could not for instance see multi-select hash tests.

Reply:
addressed comments and added test, creating new PR to branch off master since this feature branch is already merged